go.mod

```diff
@@ -42,8 +42,8 @@ require (
 	k8s.io/klog/v2 v2.30.0
 	k8s.io/kubectl v0.23.0
 	k8s.io/utils v0.0.0-20210930125809-cb0fa318a74b
-	sigs.k8s.io/cluster-api v1.1.0-beta.1
-	sigs.k8s.io/cluster-api/test v1.1.0-beta.1
+	sigs.k8s.io/cluster-api v1.1.0-rc.0
```

Comment:
the rest of the references are `1.1.0` and this one is `1.1.0-rc.0`. I think these should be updated?

Reply:
yes they should. I need the actual release to be out so I can run go modules though

Reply:
updated